llama-index-integrations/readers/llama-index-readers-wordpress/llama_index/readers/wordpress/base.py

```diff
@@ -13,7 +13,7 @@ class WordpressReader(BaseReader):
         wordpress_subdomain (str): Wordpress subdomain
     """
 
-    def __init__(self, url: str, password: str, username: str) -> None:
+    def __init__(self, url: str, password: str = None, username: str = None) -> None:
```

Comment:
nit: can you hint these types with `Optional`? Like `password: Optional[str] = None`.

Reply:
Great nit!  I learned about PEP 484 :).  Fixed.

Reply:
Great nit! I learned about PEP 484 :). Fixed.